src/PresentationalComponents/Portfolio/PortfolioActionToolbar.js

```diff
@@ -34,7 +34,7 @@ class PortfolioActionToolbar extends Component {
           <DropdownItem component="button" onClick={ this.props.onClickEditPortfolio } aria-label="Edit Portfolio">
                         Edit Portfolio
           </DropdownItem>
-          <DropdownItem component="button" aria-label="Remove Portfolio">
+          <DropdownItem component="button" onClick={ this.props.onClickRemovePortfolio } aria-label="Remove Portfolio">
                         Remove Portfolio
```

Comment:
Can you fix the indentation here please> Probably just leftover from the eslint changes.

Reply:
done